src/trans/gpu/algor/hicblas_gemm.hip.cpp

```diff
@@ -334,12 +333,14 @@ void hipblas_sgemm_wrapper_grouped(int resol_id, int blas_id, char transa,
 }
 
 void hipblas_dgemm_wrapper_grouped(int resol_id, int blas_id, char transa,
-                                   char transb, int m, int *n, int *k,
-                                   double alpha, const double *A, int lda,
-                                   int64_t *offsetsA, const double *B, int ldb,
-                                   int64_t *offsetsB, double beta, double *C,
-                                   int ldc, int64_t *offsetsC, int batchCount,
-                                   size_t stream, void *growing_allocator) {
+                                   char transb, int m, const int *n,
+                                   const int *k, double alpha, double const *A,
```

Comment:
The SGEMM equivalent has `const float *A` but here we have `double const *A`. This are different right?

Reply:
no they are the same (const applies to whatever is on the left, except there is nothing to the left, then it is to the right). but it makes sense to be consistent, will check for inconsistencies.